profiler/src/main/java/com/splunk/opentelemetry/profiler/JFR.java

```diff
@@ -61,4 +70,40 @@ public RecordedEvent readEvent(RecordingFile file, Path path) {
       throw new JfrException("Error reading events from " + path, e);
     }
   }
+
+  public void setStackDepth(int stackDepth) {
+    try {
+      JfrAccess.init();
```

Comment:
What will happen if `setStackDepth` is called multiple times (for whatever reason, but it's a public method so you never know)?